### DISCUSSION TITLE
How to format y axis data?

### DISCUSSION THREAD

**Discussion by adam-fsck** [Abhinandan-Kushwaha/react-native-gifted-charts#649] [Q&A]
Hello,

Is there any change how to format y axis data?
I am sending data like this:
[{"value": 40307}, {"value": 39831}, {"value": 39355}, {"value": 39051}, {"value": 38919}, {"value": 39246}, {"value": 39846}, {"value": 40071}, {"value": 40234}, {"value": 41030}, {"value": 40763}, {"value": 40873}, {"value": 44612}, {"value": 44184}, {"value": 44913}, {"value": 43897}, {"value": 44326}, {"value": 44476}, {"value": 44954}, {"value": 44831}, {"value": 46729}, {"value": 46736}]

But it would be nice if will be able format label to show it like this:
41 030,00 €

How can I do it, if I cannot, can u implement it somehow pls?

**Comment by arapocket** [comment]
Was this answered?
